src/test/java/teammates/test/cases/ui/InstructorCourseStudentDetailsEditSaveActionTest.java

```diff
@@ -133,7 +133,11 @@ public void testExecuteAndPostProcess() throws Exception{
         
         ______TS("Error case, invalid email parameter");
         
-        String invalidStudentEmail = "thisisaveryverylonglonglongstudentemailaccountname@gmail.tmt";
+        String invalidStudentEmail = "thisisaveryverylonglonglongveryverylonglonglong" 
+                                   + "veryverylonglonglongveryverylonglonglongveryvery"
+                                   + "longlonglongveryverylonglonglonglonglonglongveryverylonglonglong"
+                                   + "longlonglongveryverylonglonglonglonglonglongveryverlonglonglong"
+                                   + "studentemailaccountname@gmail.tmt";
```

Comment:
Ok, will change to use that method. This string is actually a boundary case of 255 characters.